canonical/vanilla-framework
# Dark theme for forms

## Done

Adds dark theme to form inputs

Fixes [WD-5201](https://warthogs.atlassian.net/browse/WD-5201)

## QA

- Open [demo](https://vanilla-framework-4857.demos.haus/docs/base/forms#dark-theme)
- Check new examples
- https://vanilla-framework-4857.demos.haus/docs/examples/patterns/forms/forms-dark
- Review updated documentation:
  - https://vanilla-framework-4857.demos.haus/docs/base/forms#dark-theme
  - https://vanilla-framework-4857.demos.haus/docs/examples/patterns/forms/form-validation-dark


### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
  - if CSS class names are not changed it can be bugfix relesase (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical/vanilla-framework/wiki/Release-process#pre-release-tasks)


## Screenshots

<img width="1207" alt="image" src="https://github.com/canonical/vanilla-framework/assets/83575/df45a771-ac88-4eb0-a389-791ed6da68d4">

<img width="1496" alt="image" src="https://github.com/canonical/vanilla-framework/assets/83575/406df5d6-c075-4e68-9186-f604a4e6ac6e">






[WD-5201]: https://warthogs.atlassian.net/browse/WD-5201?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
